### PR TITLE
[Sign-in] Add sign in button and add utils.js for saving client ID

### LINKED ISSUE #5: Sign in - [FE] social account authentication UI
make a social account sign in form

### PR DESCRIPTION
**Pull request by lsh9034**
resolves #5

### CODE REVIEW

File: web/src/components/login-form.js [hunk 1]
```diff
@@ -1,11 +1,33 @@
...
+import { clientID } from './utils.js'
 
 class LoginForm extends Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    // Google login success callback function
+    responseGoogle =  (res) => {
```
Comment (shberry) [inline web/src/components/login-form.js]: remove extra space after =

File: web/src/components/login-form.js [hunk 2]
```diff
@@ -1,11 +1,33 @@
...
+        console.log(res);
```
Comment (shberry) [inline web/src/components/login-form.js]: Add a TODO(your id): to remove this and add actual logic?

File: web/src/components/login-form.js [hunk 3]
```diff
@@ -1,11 +1,33 @@
...
+    };
+
+    // Google login fail callback function
+    responseFail = (err) => {
+        console.error(err);
```
Comment (shberry) [inline web/src/components/login-form.js]: Same here

File: web/src/components/login-form.js [hunk 4]
```diff
@@ -1,11 +1,33 @@
...
+    };
   render() {
+
```
Comment (shberry) [inline web/src/components/login-form.js]: remove extra line?

File: web/src/components/login-form.js [hunk 5]
```diff
@@ -1,11 +1,33 @@
...
     return (
         <div>
-
+            {/* Google Login Button */}
+            <GoogleLogin
+                onSuccess={this.responseGoogle}
+                onFailure={this.responseFail}
+                clientId={clientID} // our Client ID
```
Comment (shberry) [inline web/src/components/login-form.js]: Is there a reason first letter of 'Client' is capitalized?

File: web/src/components/utils.js [hunk 6]
```diff
@@ -0,0 +1,4 @@
+// this is file for sharing variables
+module.exports = {
+    clientID: "325103706675-9g44spipbpiujdkopvs8n3hvh188k14i.apps.googleusercontent.com" // Our Google API Service ID
```
Comment (shberry) [inline web/src/components/utils.js]: nit: clientId (following the naming convention, ID should be Id)

File: web/src/components/login-form.js [hunk 7]
```diff
@@ -1,11 +1,33 @@
...
+import {GoogleLogin} from 'react-google-login';
+import { clientID } from './utils.js'
 
 class LoginForm extends Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    // Google login success callback function
```
Comment (shberry) [inline web/src/components/login-form.js]: Comments should start with a verb and should be a complete sentence. Same below.

File: web/src/components/login-form.js [hunk 8]
```diff
@@ -1,11 +1,33 @@
 import React, {Component} from 'react';
 import PropTypes from 'prop-types';
+import {GoogleLogin} from 'react-google-login';
+import { clientID } from './utils.js'
```
Comment (ethkim) [inline web/src/components/login-form.js]: remove spaces before and after clientID.

File: web/src/components/utils.js [hunk 9]
```diff
@@ -0,0 +1,4 @@
+// this is file for sharing variables
```
Comment (ethkim) [inline web/src/components/utils.js]: "This is a file for sharing variables."

File: web/src/components/utils.js (hunk 6 above)
Comment (ethkim) [inline web/src/components/utils.js]: Is this the API key? Then this needs to be hidden in a secure place, not checked into the repository. 

https://www.freecodecamp.org/news/how-to-securely-store-api-keys-4ff3ea19ebda/

Now that it's already on the github commit history, we'll have to regenerate this. OK to add a TODO to fix this in a subsequent PR. Another way to do this is via environment variables and have everyone on the team set one up. 

https://gist.github.com/derzorngottes/3b57edc1f996dddcab25

**Pull request closed without merging**